src/Symfony/Component/Cache/Tests/Adapter/FilesystemAdapterTest.php

```diff
@@ -49,4 +49,62 @@ public static function rmdir($dir)
         }
         rmdir($dir);
     }
+
+    public function testPrune()
```

Comment:
can we move this to the parent test case (with a skip if the pool doesn't implement the interface)
same for PSR16?
that would require each $isHit closure to be turned into method on each specific pool's case (which I wouldn't call "ishit" btw, but maybe isPruned?)